graphql-c_parser/ext/graphql_c_parser_ext/parser.y

```diff
@@ -524,6 +524,12 @@ type_system_definition:
       /* none */      { $$ = Qnil; }
     | description
 
+  comment: STRING
```

Comment:
Ahh... I realize I missed a spot when I described the changes required here. We also need a YACC token type for `COMMENT`. 

The token types are configured in two places. First, they're laid out in an enum in lexer.rl: 

https://github.com/rmosolgo/graphql-ruby/blob/18fdf6f5e9e3b2a6d656798407aaac20857b3519/graphql-c_parser/ext/graphql_c_parser_ext/lexer.rl#L150-L206

It looks like COMMENT is already on the list there :+1: 

Then, the enum values are mapped to YACC token types here: 

https://github.com/rmosolgo/graphql-ruby/blob/18fdf6f5e9e3b2a6d656798407aaac20857b3519/graphql-c_parser/ext/graphql_c_parser_ext/parser.y#L61-L98

`COMMENT` is _not_ on the list there, since it was previously excluded from the token stream. It needs to be added. Those integer values are based on their position in the enum declaration in lexer.rl, so I _think_ we need to add `%token COMMENT 240`. 

After that, you could define this rule as `comment: COMMENT`.